src/GeneratorProxy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Gii;
+
+use Closure;
+
+final class GeneratorProxy
+{
+    private ?GeneratorInterface $generator = null;
```

Comment:
Seems, this property never used and always "null".

Reply:
It's used for memoization